source/release-notes.txt

```diff
@@ -20,6 +20,18 @@ Release Notes for |bi|
 .. expect and we don't break the links on the downloads page.
 .. (DOCS-9670 for context)
 
+.. _bi-2-14-6:
+
+|bi| 2.14.6
```

Comment:
Is the release notes specific to Atlas BIC ? If not, reporting the bug fix for [[BI-2717](https://jira.mongodb.org/browse/BI-2717)] - `Initial schema sampling occurs in 'auto' schema mode when an existing schema is available` would be great because it is a significant improvement for users of on-prem BIC